docs/deployment/rollback.md

```diff
@@ -0,0 +1,36 @@
+# Making a rollback release
+
+This list outlines the manual steps needed to make a rollback of the
+`benefits` app.
+
+If a change is deployed to the app that makes it fail to start, making a rollback
+will deploy the app to a known working state again.
+
+[Start a new Release on Github](https://github.com/cal-itp/benefits/issues/new?labels=release&template=release.yml&title=Make+a+Release){ .md-button }
+
+## 0. Create a temporary branch from the last known good release tag
```

Comment:
I thought we decided we don't need this?

Reply:
Updated the steps to match the release checklist 👍